static/js/admin.js

```diff
@@ -287,6 +295,20 @@ function _setup_page() {
         error: failed_listing_streams
     });
 
+    // Populate realm aliases
+    channel.get({
+        url: "/json/realm/aliases",
+        idempotent: true,
+        timeout: 10*1000,
+        success: function (aliases_data) {
+            update_alias_label();
+            _.each(aliases_data.aliases, function (alias) {
+                add_alias(alias);
+            });
+        },
+        error: failed_listing_aliases
+    });
```

Comment:
Ahh, OK, let's modify `page_params.domains` then.  It's basically only used for this feature, so there's little reason not to clean it up to have the interface we want :).